.gitignore

```diff
@@ -20,3 +20,9 @@ tests/docker/mongodb/storage_*
 .vscode
 distributions/docker-compose/*/storage
 venv/*
+root
+static
+tmp
+predictors
+datasources
```

Comment:
why you add this? you set in config file {storage_path} to mindsdb path?

Reply:
I usually auto-generate my configuration files and it generates them there (sometimes).

Honestly I'm not sure why all of those are there (e.g. not sure where `root`) comes from, but they were lying around, I assumed they were auto-generate by some script or another, and I didn't want to get them into the project.

Could adding any of them to gitignore cause issues ?

Reply:
No, it should not be any issues with adding this lines to gitignore. You can add it. I just surprised, how you get this folders in project root. By default it should be in `{root}/var` https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/utilities/fs.py#L23
I make fresh install in venv, run mindsdb without `--config` option, and get folder `{root}/var`.